extensions-xml-support/src/test/resources/flows/flows-with-module-using-java.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mule xmlns="http://www.mulesoft.org/schema/mule/core"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:module-using-java="http://www.mulesoft.org/schema/mule/module-using-java"
+      xsi:schemaLocation="
+       http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+       http://www.mulesoft.org/schema/mule/module-using-java http://www.mulesoft.org/schema/mule/module-using-java/current/mule-module-using-java.xsd">
+
+    <flow name="invoke-static-method-flow">
+        <module-using-java:invoke-static-method class="java.util.Collections" method="emptySet()" />
+    </flow>
+    
+    <flow name="invoke-method-flow">
+        <module-using-java:invoke-method class="java.util.HashMap" method="#[vars.method]" instance="#[vars.instance]" />
+    </flow>
+
+    <flow name="invoke-method-with-args-flow">
+        <module-using-java:invoke-method-with-args class="java.util.HashMap" method="#[vars.method]" args="#[vars.args]" instance="#[vars.instance]" />
+    </flow>
+
+    <flow name="new-method-flow">
+        <module-using-java:new-method class="java.util.HashMap" constructor="HashMap()" />
```

Comment:
this seems redundant with `new-method-custom-class-flow`

Reply:
Both test flows (with and without `...-custom-class`) are there to show not only java libraries (HashMap) are accesible, but also compiled classes within the connector (probably most common use case when using java module thru a smart connector)